packages/instantsearch.js/package.json

```diff
@@ -55,9 +55,9 @@
     "version": "./scripts/version/update-version.js"
   },
   "devDependencies": {
-    "@instantsearch/mocks": "1.26.0",
-    "@instantsearch/tests": "1.26.0",
-    "@instantsearch/testutils": "1.15.0",
+    "@instantsearch/mocks": "1.27.0",
+    "@instantsearch/tests": "1.27.0",
+    "@instantsearch/testutils": "1.16.0",
```

Comment:
something is still depending on 1.15.0 🤔

Reply:
I updated this and other versions mismatchs in the latest commit. I'll let you re-review it.

Reply:
It happened last time too, wonder what's wrong in the diff checking